src/providers/google-login-provider.ts

```diff
@@ -9,18 +10,15 @@ export class GoogleLoginProvider extends BaseLoginProvider {
 
   protected auth2: any;
 
-  constructor(private clientId: string) { super(); }
+  constructor(private clientId: string, private opt: LoginOpt = { client_id: clientId, scope: 'email'}) { super(); }
```

Comment:
The `clientId` is already present as the first parameter. Why do you need it in the second parameter too?

Reply:
yeah i've leave up the 1st params unused in this situation.
I've change to rest params to make it work, please refer newest commit